Cargo.toml

```diff
@@ -7,3 +7,11 @@ keywords = ["texture", "atlas", "bin", "box", "packer"]
 description = "A general purpose, deterministic bin packer designed to conform to any two or three dimensional use case."
 license = "MIT/Apache-2.0"
 repository = "https://github.com/chinedufn/rectangle-pack"
+
+[features]
+default = ["std"]
+std = []
+
+[target.'cfg(not(feature = "std"))'.dependencies]
```

Comment:
I'd love it if we could avoid introducing the extra dependency and just use the `core` crate. The `core` crate should work regardless of whether or not we are in `no_std`.

Since this crate is fairly small it hopefully isn't a hassle to just convert all of the `std` -> `core`.

Reply:
The reason I added it was because I couldn't figure out how to do `use std::cmp::Ordering` - do you have any idea what to do for this?

Reply:
actually the core seemed to work for that one, now I'm having trouble with Hash